adcpipeline/pipeline.py

```diff
@@ -1,14 +1,13 @@
+import logging
 import os
 import re
-import logging
-import pandas as pd
 from abc import ABC
 from typing import Callable, Dict, List, Iterator, Optional
 
+import pandas as pd
```

Comment:
Nit picking: I like the imports at the top of the file, not between the 'from' import statements

Reply:
I have auto formatting on in PyCharm, so apparently this is just PEP8 style. I think it always makes sections for python "native" packages (abc and typing for instance) and pip installed packages (pandas in this case) and _then_ I think it makes a distinction between imports and from imports. I'll change it for you if I need to other updates, otherwise I'll just leave it for now and you can change it next time ;)